src/js/game/LevelMVC/LevelView.js

```diff
@@ -2280,8 +2289,16 @@ module.exports = class LevelView {
         xOffset = this.blocks[blockType][2];
         yOffset = this.blocks[blockType][3];
         sprite = group.create(xOffset + 40 * x, yOffset + group.yOffset + 40 * y, atlas, frame);
-        frameList = Phaser.Animation.generateFrameNames("Conduit", 3, 10, "", 2);
+        frameList = Phaser.Animation.generateFrameNames("Conduit", 0, 0, "", 2);
         sprite.animations.add("idle", frameList, 5, true);
```

Comment:
Think we can...... the suspense!

I'm reading this as "think we can get rid of these two lines"? That makes sense as long as we get that default first frame, since it's not actually an animation... Which is absolutely what `"conduit": ["blocks", "Conduit00", -12, 0],` is doing _huh_. Whoopse! I had forgotten what those strings/ints were all about. Haha